components/Card/Card.styles.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  BookmarkSquareIcon, CheckBadgeIcon, ClockIcon, UserCircleIcon, 
+} from '@heroicons/react/24/solid';
+
+export const iconMap = {
+  user: UserCircleIcon,
+  badge: CheckBadgeIcon,
+  clock: ClockIcon,
+  bookmark: BookmarkSquareIcon,
+};
+export type IconType = keyof typeof iconMap;
+
+/**
+ * Normalized base size and position of each icon (finetuned manually) for use in eg, buttons
+ * Only add to this map if different from default class w-1em
+ * If you wish to use the HeroIcon without any base styles, set the noBaseStyle boolean prop to true
+ */
+
+// This basically means that the keys from iconBaseStyle are from the keys of iconMap
+type IconBaseStyleType = Partial<{
+  [Key in IconType]: string;
+}>;
+
+export const iconBaseStyleDefault = 'w-1em';
+export const iconBaseStyle: IconBaseStyleType = {
+  user: 'w-60',
+  badge: 'w-60',
+  clock: 'w-60',
+  bookmark: 'w-60',
+};
```

Comment:
I would set iconBaseStyleDefault to w-60 instead of w-1em, and you can probably modify things here to get rid of the individual widths for each icon here since they're the same as the default. This is how I did it on Campaign, ie, no need to add individual base style if they're the same as the default.